superset/views/core.py

```diff
@@ -7,18 +7,18 @@
 from datetime import datetime, timedelta
 import json
 import logging
-import pandas as pd
 import pickle
 import re
 import time
 import traceback
-
+import os
```

Comment:
NIT: when there are many imports in a module, standard lib import should be together in a group (empty line between import groups), and alpha-sorted within that group preferably 

organizing imports becomes more important as there are more imports in the module

Reply:
Fixed